.circleci/config.yml

```diff
@@ -63,9 +60,68 @@ jobs:
             source env/bin/activate
             python manage.py test
 
+  browser-test:
+    docker:
+      - image: circleci/python:2.7-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Setup Python environment
+          command: |
+            pip install virtualenv
+            virtualenv env
+
+      - restore_cache:
+          keys:
+            - pip-v2-{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/test.txt" }}
+      - run:
+          name: Install dependencies
+          command: |
+            source env/bin/activate
+            pip install --requirement requirements.txt --requirement requirements/test.txt
+      - save_cache:
+          key: pip-v2-{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/test.txt" }}
+          paths:
+            - "~/.cache/pip"
+
+      - restore_cache:
+          keys:
+            - npm-v1-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install javascript dependencies
+          command: |
+            npm install
+            ./node_modules/.bin/bower install
+            npm run update-selenium
+      - save_cache:
+          key: npm-v1-{{ checksum "package-lock.json" }}
+          paths:
+            - "./node_modules"
+
+      - run:
+          name: Compile assets
+          command: ./node_modules/.bin/gulp
```

Comment:
If this were an npm script, we could change this line to something like:

`npm run build`

I think it'd make it easier to understand how to build the assets from a new developer perspective.

Reply:
Ah, okay.

On reflection, I think the work to simplify the installation of front-end dependencies and building of assets could be done in separate piece of work - we want to remove bower anyway.